doc/cephfs/fs-volumes.rst

```diff
@@ -766,6 +766,14 @@ Will enable distributed subtree partitioning policy for the "csi" subvolume
 group.  This will cause every subvolume within the group to be automatically
 pinned to one of the available ranks on the file system.
 
+To disable a partioning policy, set `pin_setting` to 0:
+
+.. prompt:: bash #
+
+   ceph fs subvolumegroup pin cephfilesystem-a csi distributed 0
```

Comment:
@batrick Looks to me that unsetting the export pin using mgr/volumes interface will lead to pinning the group to rank 0 as pin_util.py does

```
    try:
        pin_setting = _pin_value[pin_type](pin_setting)
    except ValueError as e:
	raise VolumeException(-errno.EINVAL, f"pin value wrong type: {pin_setting}")
```

Reply:
Sorry, I'm confused. Wouldn't you use "-1" to disable an export pin? (Also, this section of the document is referring to distributed ephemeral pins?)

Reply:
Yes, that's right, however, I have always removed the export pin xattr rather than setting it to `-1 and therefore the source of confusion came up.

Reading this part of the doc, it might also feel that setting `pin_setting` to 0 (for export pin) will disable the export pin. I feel that part needs to be made clear.

Reply:
Btw, the -1 set is mentioned in the multimds doc, but I don't expect that (some) users would cross refer to that for the value to be used for `pin_setting` to disable export pin.